samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/AssessmentSettingsBean.java

```diff
@@ -1773,7 +1774,7 @@ public void saveEditedExtendedTime() {
       Integer oldIndex = null;
       // is there a more efficient way?
       for(ExtendedTime entry : extendedTimes) {
-        if(entry.getId().equals(this.extendedTime.getId())) {
+        if(ObjectUtils.equals(entry.getId(), this.extendedTime.getId())) {
```

Comment:
I believe you made this change as it is null safe, good catch!

For future reference, [ObjectUtils.equals](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/ObjectUtils.html#equals-java.lang.Object-java.lang.Object-) is deprecated.